cmd/ocm-backplane/cloud/assume.go

```diff
@@ -17,14 +18,20 @@ import (
 )
 
 const (
-	DefaultInitialRoleArn = "arn:aws:iam::922711891673:role/SRE-Support-Role"
+	InitialRoleArnTemplate             = "arn:aws:iam::%v:role/SRE-Support-Role"
+	EnvProd                Environment = "prod"
+	ProdPayerAccountID                 = 922711891673
+	EnvStg                 Environment = "stg"
+	StgPayerAccountId                  = 277304166082
+	EnvInt                 Environment = "int"
+	IntPayerAccountId                  = 277304166082
```

Comment:
What if moving the payerAccountID into the backplane config file [~/.config/backplane] ?
Otherwise, this exposes the payer accountID in a public repo for all envs .
There may be more supported env soon, [eg: `fedramp`], and this approach may need to change then.

Reply:
I spoke about this with @jharrington22 way back when we began this work and we determined that [the ARNs aren't secret so there's no issue with having them in GitHub](https://redhat-internal.slack.com/archives/C058T1YEDPV/p1684869351960399?thread_ts=1684868454.749649&channel=C058T1YEDPV&message_ts=1684869351.960399).

Would you still like me to move these to a config file?